src/app/templates/CreatePasswordModal/index.tsx

```diff
@@ -85,6 +90,7 @@ export const CreatePasswordModal = memo<CreatePasswordModalProps>(
     const onSubmit = useCallback(
       async (data: FormData) => {
         // TODO: enable onboarding when it is reimplemented
```

Comment:
Should we remove this TODO comment or is it still relevant?

Reply:
No, it's still relevant